test/test_artifacts/v1/jupyter-server-fileid.test.Dockerfile

```diff
@@ -0,0 +1,6 @@
+ARG COSMOS_IMAGE
+FROM $COSMOS_IMAGE
+
+ARG MAMBA_DOCKERFILE_ACTIVATE=1
+
+CMD ["python", "-c", "import jupyter_server_fileid"]
```

Comment:
Can we add more testing than just import? Test soms functionality if possible.

Reply:
The jupyter_server_fileid is a Jupyter server extension which requires Jupyter server to run and test. I would prefer having it run as integ test on an actual JL app, if make sense